docs/optimizations/CheckpointFiles.md

```diff
@@ -4,27 +4,28 @@
 
 Available in Cromwell 55 and higher.
 
-This optimization hopes to resolve the issue of your worker VM being preempted 9 hours and 55 minutes into the runtime of
+This optimization aims to resolve the issue of your worker VM being preempted 9 hours and 55 minutes into the runtime of
 a 10 hour job and having no option but to re-run the entire computation again.
 
 ### Description
 
 Specifying a `checkpointFile` value in a task's `runtime` section designates a checkpoint file which will periodically be
-copied to cloud storage every 10 minutes.
-This checkpoint file will then be restored automatically on subsequent attempts if the job is interrupted.
+copied to cloud storage every 10 minutes. This checkpoint file will then be restored automatically on subsequent attempts if the job is interrupted.
 
-**Note:** Although the checkpoint file is deleted if the task succeeds, additional charges may accrue storing the checkpoint file during
-the running of the task, if the task is aborted or otherwise stopped externally, and by transferring it between the VM and the cloud. These
+To use this feature effectively, the WDL task must be written intentionally to use the checkpoint file. See example below. 
+
+**Note:** Although the checkpoint file is deleted if the task succeeds, storage charges may accrue from (1) storing the checkpoint file during
+the running of the task, (2) aborting or otherwise stopping the task externally, and (3) transferring it between the VM and the cloud storage bucket. These
 cost should be minor, especially balanced against the performance and cost benefits of being able to restore from the
-checkpoint when preemptible VMs are interrupted.   
+checkpoint when a worker VMs gets preempted.   
 
 ### Effect on Call Caching
 
 The presence or absence of the `checkpointFile` attribute is not considered when determining whether to call cache.  
 
 ### Example
 
-The following WDL demonstrates the use of the `checkpointFile` optimization. It has a command which is checkpoint-aware:
+The following WDL demonstrates the use of the `checkpointFile` optimization. It has a command that is checkpoint-aware:
```

Comment:
Counter-nit: IMO this sub-clause is adding more information about the already-specified "command" rather than requiring the sub-clause _in order to_ specify which of many "command"s is being talked about.

Reply:
Hah, well this is your queen's language so I may be out of my lane; "which" just looked wrong to me. I have whatever is the equivalent of synesthesia for grammar… 

(… but I still think it's a defining clause even if it's not required for comprehension)

Reply:
👍  I think "that" in this case is fine. Happy for you to leave this as you currently have it.